src/ResourceManager/Profile/Commands.Profile/Environment/AddAzureRMEnvironment.cs

```diff
@@ -146,6 +146,11 @@ public string DataLakeAudience
             }
         }
 
+        [Parameter(Position = 20, Mandatory = false, ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource identifier of the Azure Batch service that is the recipient of the requested token.")]
+        [Alias("BatchResourceId")]
```

Comment:
I think we want to use BatchAudience as well

Reply:
As an alias you mean? Will add it.

Reply:
This is done